doc/changelogs/CHANGELOG_V18.md

```diff
@@ -0,0 +1,286 @@
+# Node.js 18 ChangeLog
+
+<!--lint disable maximum-line-length no-literal-urls prohibited-strings-->
+
+<table>
+<tr>
+<th>Current</th>
+</tr>
+<tr>
+<td>
+<a href="#18.0.0">18.0.0</a><br/>
+</td>
+</tr>
+</table>
+
+* Other Versions
+  * [17.x](CHANGELOG_V17.md)
+  * [16.x](CHANGELOG_V16.md)
+  * [15.x](CHANGELOG_V15.md)
+  * [14.x](CHANGELOG_V14.md)
+  * [13.x](CHANGELOG_V13.md)
+  * [12.x](CHANGELOG_V12.md)
+  * [11.x](CHANGELOG_V11.md)
+  * [10.x](CHANGELOG_V10.md)
+  * [9.x](CHANGELOG_V9.md)
+  * [8.x](CHANGELOG_V8.md)
+  * [7.x](CHANGELOG_V7.md)
+  * [6.x](CHANGELOG_V6.md)
+  * [5.x](CHANGELOG_V5.md)
+  * [4.x](CHANGELOG_V4.md)
+  * [0.12.x](CHANGELOG_V012.md)
+  * [0.10.x](CHANGELOG_V010.md)
+  * [io.js](CHANGELOG_IOJS.md)
+  * [Archive](CHANGELOG_ARCHIVE.md)
+
+<a id="18.0.0"></a>
+
+## 2022-04-19, Version 18.0.0 (Current), @BethGriggs
+
+Node.js 18 is here! Highlights include the update of the V8 JavaScript engine to 10.1, global fetch enabled by default, and a core test runner module.
+
+Initially, Node.js 18 will replace Node.js 17 as our ‘Current’ release line. As per the release schedule, Node.js 18 will be the ‘Current’ release for the next 6 months and then promoted to Long-term Support (LTS) in October 2022. Once promoted to long-term support the release will be designated the codename ‘Hydrogen’. Node.js 18 will be supported until April 2025.
+
+### Notable Changes
+
+#### Deprecations and Removals
+
+* \[[`0bac5478eb`](https://github.com/nodejs/node/commit/0bac5478eb)] - **(SEMVER-MAJOR)** **fs**: runtime deprecate string coercion in `fs.write`, `fs.writeFileSync` (Livia Medeiros) [#42607](https://github.com/nodejs/node/pull/42607)
+* \[[`08773e3c04`](https://github.com/nodejs/node/commit/08773e3c04)] - **(SEMVER-MAJOR)** **dns**: remove `dns.lookup` and `dnsPromises.lookup` options type coercion (Antoine du Hamel) [#41431](https://github.com/nodejs/node/pull/41431)
+* \[[`60b8e79599`](https://github.com/nodejs/node/commit/60b8e79599)] - **(SEMVER-MAJOR)** **process**: runtime deprecate multipleResolves (Benjamin Gruenbaum) [#41896](https://github.com/nodejs/node/pull/41896)
+* \[[`58e645de63`](https://github.com/nodejs/node/commit/58e645de63)] - **(SEMVER-MAJOR)** **stream**: remove thenable support (Robert Nagy) [#40773](https://github.com/nodejs/node/pull/40773)
+* \[[`807c7e14f4`](https://github.com/nodejs/node/commit/807c7e14f4)] - **(SEMVER-MAJOR)** **tls**: move tls.parseCertString to end-of-life (Tobias Nießen) [#41479](https://github.com/nodejs/node/pull/41479)
+
+#### fetch (experimental)
+
+In Node.js 18, an experimental global fetch API is available by default. The implementation is based upon [undici](https://undici.nodejs.org/#/), an HTTP/1.1 client written for Node.js by contributors to the project.
+
+```mjs
+const res = await fetch('https://example.com');
+console.log(res);
```

Comment:
```suggestion
if (res.ok) {
  const data = await res.json();
  console.log(data);
}
```

Reply:
Any suggestions for an example endpoint that returns JSON? While in intent is inferable for us - I'd rather the snippet work for anyone who just copy/pastes.

Reply:
https://nodejs.org/api/fs.json ?

Reply:
Ah, I didn't think of using one of ours. Might go for https://nodejs.org/api/documentation.json as it's much smaller?

Reply:
lol both of those display wrong to my mobile browser because they end in “.json”

Reply:
> Might go for https://nodejs.org/api/documentation.json as it's much smaller?

I'm fine either way, I've suggested `fs.json` because it makes the URL shorter. `index.json` would also work.